src/models/Models.jl

```diff
@@ -63,14 +77,18 @@ function fit_model(counterfactual_data::CounterfactualData, model::Symbol=:MLP;
 
     # Set up:
     M = all_models_catalogue[model](counterfactual_data; kwrgs...)
-
-    # Train:
     train(M, counterfactual_data)
 
     return M
 end
 
 export standard_models_catalogue,
```

Comment:
Exports should be next to each other in a single line, only making additional lines if required. Look under "Function Exports" here - https://github.com/invenia/BlueStyle.

Reply:
Fixed, thanks!